packages/appstore/src/assets/svgs/wallets/index.ts

```diff
@@ -0,0 +1,3 @@
+import WalletConsentFormIcon from './ic-wallets-consent-form.svg';
```

Comment:
use the `WalletsIcon`  component in @sergei-deriv 's [PR](https://github.com/binary-com/deriv-app/blob/5e39ca9a87826ec3ba88643e2ec0f8d964cafeb1/packages/appstore/src/assets/svgs/wallets/index.tsx)

Reply:
Done 😃 👍🏼